lib/server.js

```diff
@@ -207,8 +208,12 @@ Server.prototype.stop = function (callback) {
     self._err = codeOrErr instanceof Error ? codeOrErr : self._err;
     callback(self._err);
   });
-
   proc.on("exit", _done);
+
+  // Kill it the "official" way.
   proc.kill();
+
+  // Kill it the unofficial way.
+  treeKill(proc.pid, "SIGKILL", _done);
```

Comment:
Are there any consequences of sending such a strong kill signal? Will the process clean up everything synchronously thru the regular `proc.kill()`?